### PR TITLE
fix: Stop sending not specified optional fields when creating a user

### PR DESCRIPTION
**Pull request by lukaszsocha2**
Closes: SDK-2589

### CODE REVIEW

File: src/main/java/com/box/sdk/CreateUserParams.java [hunk 1]
```diff
@@ -28,7 +29,7 @@ public class CreateUserParams {
      *
      * @return true if the new user will be able to see other enterprise users in their contact list; otherwise false.
      */
-    public boolean getCanSeeManagedUsers() {
+    public Boolean getCanSeeManagedUsers() {
```
Comment (antusus) [inline src/main/java/com/box/sdk/CreateUserParams.java:32]: This is a breaking change in the API we have to mark it in the PR title

**Review by antusus — CHANGES_REQUESTED** [review]
We are making a breaking change in the API because of the CreateUserParams are now returning null. We have to show it in the title

File: src/main/java/com/box/sdk/CreateUserParams.java (hunk 1 above)
Comment (lukaszsocha2) [reply src/main/java/com/box/sdk/CreateUserParams.java:32]: I consider it as a small breaking change (Usually clients won't use getters for an object used only to make a request). Would adding `!` not cause bumping major version ?

**Review by antusus — APPROVED** [review]
👍

**Review by antusus — DISMISSED** [review]
👍

**Review by antusus — APPROVED** [review]
👍

**Pull request merged**